docs/index.md

```diff
@@ -8,46 +8,89 @@ description: |-
+If you would prefer to provide an access token directly, you can do so with the `access_token` attribute, or the corresponding environment variable. For information on obtaining an access token to use with this provider, see the [documentation](https://docs.pingidentity.com/pingoneaic/latest/developer-docs/authenticate-to-rest-api-with-access-token.html).
 
-```hcl
+```terraform
 provider "identitycloud" {
   tenant_environment_fqdn = var.my_aic_tenant_environment_fqdn
   access_token            = var.my_aic_access_token
 }
 ```
 
-<!-- schema generated by tfplugindocs -->
-## Schema
+### Custom User Agent information
+
+This Terraform provider allows custom information to be appended to the default user agent string (that includes Terraform provider version information) by setting the `PINGONE_AIC_TF_APPEND_USER_AGENT` environment variable.  This can be useful when troubleshooting issues with Ping Identity Support, or adding context to HTTP requests.
 
-### Required
+```shell
+export PINGONE_AIC_TF_APPEND_USER_AGENT="Jenkins/2.426.2"
+```
 
-- `tenant_environment_fqdn` (String, Sensitive) The fully qualified domain name of the tenant environment. Default value can be set with the `PINGAIC_TF_TENANT_ENV_FQDN` environment variable.
+<!-- schema generated by tfplugindocs -->
+## Schema
 
 ### Optional
 
 - `access_token` (String, Sensitive) Access token for the PingOne Advanced Identity Cloud Rest API. Default value can be set with the `PINGAIC_TF_ACCESS_TOKEN` environment variable. If `access_token` is provided, `service_account_id` and `service_account_private_key` should not be provided.
 - `service_account_id` (String, Sensitive) Service account ID for the PingOne Advanced Identity Cloud Rest API. The service account must have the following scopes: `fr:idc:certificate:*`, `fr:idc:content-security-policy:*`, `fr:idc:cookie-domain:*`, `fr:idc:custom-domain:*`, `fr:idc:esv:*`, `fr:idc:promotion:*`, `fr:idc:sso-cookie:*`. Default value can be set with the `PINGAIC_TF_SERVICE_ACCOUNT_ID` environment variable. If `service_account_id` and `service_account_private_key` are provided, `access_token` should not be provided.
 - `service_account_private_key` (String, Sensitive) Service account private key for the PingOne Advanced Identity Cloud Rest API. Default value can be set with the `PINGAIC_TF_SERVICE_ACCOUNT_PRIVATE_KEY` environment variable. If `service_account_id` and `service_account_private_key` are provided, `access_token` should not be provided.
+- `tenant_environment_fqdn` (String, Sensitive) The fully qualified domain name of the tenant environment. Default value can be set with the `PINGAIC_TF_TENANT_ENV_FQDN` environment variable.
```

Comment:
This is technically optional in that the terraform value isn't required, but only because you can use the environment variable instead. In effect it is required - either the provider attribute or the env var must be provided. That's why I had it moved into Required, and didn't use the default SchemaMarkdown in the template.